util/dockerfiles/README.md

```diff
@@ -66,7 +66,7 @@ Hello, world!
 # Documentation
 
 Chapel's documentation is [available online](https://chapel-lang.org/docs/).
-Documentation for a specific release is also available: [1.19](https://chapel-lang.org/docs/1.19/).
+Documentation for a specific release is also available: [1.22](https://chapel-lang.org/docs/1.22/).
```

Comment:
Seems like this should point to 1.23

Reply:
This is giving an example of a specific release doc aka the previous version.  The 1.23 is the latest and is the default link above.